core/src/main/java/apoc/export/json/JsonFormat.java

```diff
@@ -49,11 +50,9 @@ private ProgressInfo dump(Writer writer, Reporter reporter, Consumer<JsonGenerat
             consumer.accept(jsonGenerator);
             jsonGenerator.flush();
             tx.commit();
-            reporter.done();
-            return reporter.getTotal();
-        } finally {
-            writer.close();
         }
```

Comment:
Is it safe to remove the `write.close()`? Keeping it but still moving the two reporter lines will also solve the issue.

Reply:
The `writer.close()` should be unnecessary, because the `WriterBasedJsonGenerator.close()`, called by the `getJsonGenerator(writer)`, has a `writer.close();`, so we can remove it because of try-with-resource.

Reply:
Ah ok, sounds like it should be fine then :) Just wanted to check so it wasn't unintentional